log4j.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd" >
+<log4j:configuration>
+    <appender name="stdout" class="org.apache.log4j.ConsoleAppender">
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%d{ABSOLUTE}
+      %5p %c{1}:%L - %m%n"/>
+        </layout>
+    </appender>
+    <appender name="daniel" class="org.apache.log4j.FileAppender">
```

Comment:
That appender is not in the original OxGarage and looks like some temporary debugging code? If we want to keep it though, we should change the name "daniel" to something more meaningful?!

Reply:
Yes, but this is the version the configuration was in for MEIGarage before modularizing